[2024-01-17T22:57:36.420+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stream_Data_5.coinMarket scheduled__2024-01-17T22:56:27.104619+00:00 [queued]>
[2024-01-17T22:57:36.431+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stream_Data_5.coinMarket scheduled__2024-01-17T22:56:27.104619+00:00 [queued]>
[2024-01-17T22:57:36.431+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T22:57:36.446+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): coinMarket> on 2024-01-17 22:56:27.104619+00:00
[2024-01-17T22:57:36.451+0000] {standard_task_runner.py:57} INFO - Started process 8554 to run task
[2024-01-17T22:57:36.455+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Stream_Data_5', 'coinMarket', 'scheduled__2024-01-17T22:56:27.104619+00:00', '--job-id', '1215', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream_5.py', '--cfg-path', '/tmp/tmp5jzdnuhm']
[2024-01-17T22:57:36.458+0000] {standard_task_runner.py:85} INFO - Job 1215: Subtask coinMarket
[2024-01-17T22:57:36.540+0000] {task_command.py:415} INFO - Running <TaskInstance: Stream_Data_5.coinMarket scheduled__2024-01-17T22:56:27.104619+00:00 [running]> on host 614a419ad930
[2024-01-17T22:57:36.679+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='Stream_Data_5' AIRFLOW_CTX_TASK_ID='coinMarket' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T22:56:27.104619+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T22:56:27.104619+00:00'
[2024-01-17T22:57:36.737+0000] {logging_mixin.py:151} INFO - HTTPSConnectionPool(host='pro-api.coinmarketcap.com', port=443): Max retries exceeded with url: /v1/cryptocurrency/listings/latest?start=1&convert=USD (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f33e0806c40>: Failed to resolve 'pro-api.coinmarketcap.com' ([Errno -2] Name or service not known)"))
[2024-01-17T22:57:36.737+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T22:57:36.753+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Stream_Data_5, task_id=coinMarket, execution_date=20240117T225627, start_date=20240117T225736, end_date=20240117T225736
[2024-01-17T22:57:36.790+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T22:57:36.828+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
